docs/docs/how-tos/debug-nebari.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: debug-nebari
+title: Debug Nebari deployment
+description: Debug your Nebari deployment
+---
+
+## Debug the Kubernetes Cluster
+
+If you need more information about your Kubernetes cluster, [`k9s`](https://k9scli.io/) is a terminal-based UI that is extremely useful for debugging. It simplifies navigating, observing, and managing your applications in Kubernetes. `k9s` continuously monitors Kubernetes clusters for changes and provides shortcut commands to interact with the observed resources. It's a fast way to review and resolve day-to-day issues in Kubernetes, a huge improvement to the general workflow, and a best-to-have tool for debugging your Kubernetes cluster sessions.
+
+[Installation instructions for macOS, Windows, and Linux](https://github.com/derailed/k9s) are available.
+
+By default, `k9s` starts with the standard directory that's set as the context (in this case Minikube). To view all the current process press 0:
```

Comment:
The second sentence is a subject/verb disagreement: "To view all the current process**es** press `0`:"